python/pyarrow/tests/test_substrait.py

```diff
@@ -145,3 +145,8 @@ def test_binary_conversion_with_json_options(tmpdir):
     res_tb = reader.read_all()
 
     assert table.select(["bar"]) == res_tb.select(["bar"])
+
+
+def test_get_supported_functions():
+    supported_functions = pa._substrait.get_supported_functions()
+    assert len(supported_functions) > 0
```

Comment:
Is it wise to assert the supported functions by name rather than checking if it returns a `len > 0` list?